From 9d99614b886d9d7a957a0751b4287cf1579423f3 Mon Sep 17 00:00:00 2001
From: WolfEnders <grzwolf1@gmail.com>
Date: Sat, 17 Nov 2018 17:09:25 +0100
Subject: [PATCH] Bump Security Patch Level to 2018-10-01

Implemented Patches
===================
CVE-2018-9492   A-111934948     High        231156
CVE-2018-9493   A-111085900     High        231159
CVE-2018-9504   A-110216176     Critical    231594
CVE-2018-9502   A-111936792     High        231597
CVE-2018-9502   A-111936792 [2]             231614
CVE-2018-9502   A-111936792 [3]             231615
CVE-2018-9503   A-80432928      High        231597
CVE-2018-9503   A-80432928  [2]             231614
CVE-2018-9503   A-80432928  [3]             231615
CVE-2018-9505   A-110791536     High        231616
CVE-2018-9506   A-111803925     High        231621
CVE-2018-9507   A-111893951     High        231622
CVE-2018-9509   A-111937027     High        231678

Skipped Patches
===============
CVE-2018-9490   A-111274046     Critical    not applicable in KK
CVE-2018-9490   A-111274046 [2]             not applicable in KK
CVE-2018-9491   A-111603051     High        NdkMediaCodec.cpp not existing in KK
CVE-2018-9493   A-111085900 [2]             not applicable in KK
CVE-2018-9493   A-111085900 [3]             not applicable in KK
CVE-2018-9452   A-78464361      Moderate    class HorizontalMeasurementProvider not existing
CVE-2018-9452   A-78464361  [2]             not applicable in KK
CVE-2018-9473   A-65484460      Critical    ihevcd_parse_headers.c not existing
CVE-2018-9496   A-110769924     Critical    ixheaacd* files not existing
CVE-2018-9497   A-74078669      Critical    impeg2d* files not existing
CVE-2018-9498   A-78354855      Critical    not applicable in KK
CVE-2018-9499   A-79218474      High        class member BnCrypto::readVector does not exist
CVE-2017-13283  A-78526423      Critical    avrc_bld_ct.cc is too much deviated compared to KK avrc_bld_ct.c
CVE-2018-9476   A-109699112     Critical    btif_av.cc is too much deviated compared to KK btif_av.c
CVE-2018-9501   A-110034419     High        not applicable in KK
CVE-2018-9508   A-111936834     High        no method smp_process_keypress_notification in smp_act.c
CVE-2018-9510   A-111937065     High        method smp_command_has_invalid_parameters in smp_utils.c missing
CVE-2018-9511   A-111650288     High        XfrmController.cpp not existing in KK

Change-Id: Ibedc99dfce9c0bc59edf6d502a9bab5f0132d9f2
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 77ebba4..41b4a03 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -81,7 +81,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
   # Can be an arbitrary string, but must be a single word.
   #
   # If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-  PLATFORM_SECURITY_PATCH := 2018-09-01
+  PLATFORM_SECURITY_PATCH := 2018-10-01
 endif
 
 ifeq "" "$(BUILD_ID)"
-- 
2.7.4

